CONTRIBUTING.md

```diff
@@ -87,7 +87,6 @@ Please contribute! The Layer5 site uses Gatsby. The process of contributing to d
 `vi <specific page>.md` # or use your favorite IDE
 1. Run site locally to preview changes.
 `make site` # this will run a local web server with "live reload" conveniently enabled.
-(NOTE make is a linux command , if you are using windows you can install wsl , and then run command `sudo apt install make` )
```

Comment:
Maybe we can add, that the `make` command sometimes gives issues on the windows machine as the PATH for the binary might not be set correctly. So, double-check that, if you see any errors on using the `make` command like `unrecognized command`.
Or, you can frame it in a better way 😅

Reply:
Okay i'll try 😅